cli-helpers/src/commands/invoke-avl-datafeed-validator.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from "@commander-js/extra-typings";
+import inquirer from "inquirer";
+import { STAGES, STAGE_OPTION_WITH_DEFAULT, invokeLambda } from "../utils";
+
+export const invokeAvlDataFeedValidator = new Command("invoke-avl-data-feed-validator")
+    .addOption(STAGE_OPTION_WITH_DEFAULT)
```

Comment:
```suggestion
    .addOption(STAGE_OPTION)
```

In order for the inquirer prompt below to be usable